environments/staging/inventory.ini.j2

```diff
@@ -105,10 +105,12 @@ redis2
 
 [elasticsearch:children]
 # Amazon EC2
-es4
-es5
-#es6 this is intentional until testing is over, please contact sravan before modifying these
-#es7
+# old ES v2 hosts
+# es4
+# es5
```

Comment:
Can these lines be deleted (rather than commented)?

Reply:
We can delete them once QA is pass.